Index: libwacom/meson.build
===================================================================
--- libwacom.orig/meson.build
+++ libwacom/meson.build
@@ -236,6 +236,8 @@ data_files = [
 	'data/isdv4-10f.tablet',
 	'data/isdv4-12c.tablet',
 	'data/isdv4-48c9.tablet',
+	'data/isdv4-48ca.tablet',
+	'data/isdv4-48ce.tablet',
 	'data/isdv4-48ec.tablet',
 	'data/isdv4-48ed.tablet',
 	'data/isdv4-48ee.tablet',
@@ -266,6 +268,7 @@ data_files = [
 	'data/isdv4-51bf.tablet',
 	'data/isdv4-51c4.tablet',
 	'data/isdv4-51e2.tablet',
+	'data/isdv4-51e9.tablet',
 	'data/isdv4-51f5.tablet',
 	'data/isdv4-51f6.tablet',
 	'data/isdv4-90.tablet',
